governance/xc_admin/packages/xc_admin_cli/src/index.ts

```diff
@@ -74,7 +75,7 @@ async function loadVaultFromOptions(options: any): Promise<MultisigVault> {
   const vault: PublicKey = new PublicKey(options.vault);
 
   const squad = SquadsMesh.endpoint(
-    getPythClusterApiUrl(multisigCluster),
+    options.rpcUrlOverride || getPythClusterApiUrl(multisigCluster),
```

Comment:
It's generally better to use `??` for these conditionals.

Reply:
Yes